cmd/podman/port.go

```diff
@@ -125,12 +126,14 @@ func portCmd(c *cliconfig.PortValues) error {
 			if v.ContainerPort == int32(userPort) {
 				if userProto == "" || userProto == v.Protocol {
 					fmt.Printf("%s:%d\n", hostIP, v.HostPort)
+					found = true
 					break
 				}
-			} else {
-				return errors.Errorf("No public port '%d' published for %s", userPort, containerName)
 			}
 		}
+		if !found && port != "" {
```

Comment:
Do we really care if port is empty here?  If so, should we just skip this for loop entirely if port is blank?